File: DAOLib/Models/InvestmentProducts.cs
using System;
using System.ComponentModel;
using System.ComponentModel.DataAnnotations;

namespace DAOLib.Models
{
    public class InvestmentProducts
    {
        [Key]
        public virtual int id { set; get; }

        public virtual String? description { set; get; }

        public virtual String? symbol { set; get; }

        public virtual String? url { set; get; }
        
        [DefaultValue(true)]
        public virtual bool? active { set; get; }
    }
}

File: WebApp/GARCA_API/app/Models/InvestmentProducts.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Model;
use Illuminate\Database\Eloquent\Factories\HasFactory;

class InvestmentProducts extends Model
{
    use HasFactory;

    protected $table = 'investment_products';

    protected $fillable = ['description', 'investmentProductsTypesId', 'symbol', 'url', 'active'];
}
